sql/core/src/test/resources/sql-tests/inputs/postgreSQL/interval.sql

```diff
@@ -156,15 +156,6 @@
 -- [SPARK-29389] Support synonyms for interval units
 -- select '100000000y 10mon -1000000000d -100000h -10min -10.000001s ago'::interval;
 
--- test justify_hours() and justify_days()
--- [SPARK-29390] Add the justify_days(), justify_hours() and justify_interval() functions
```

Comment:
seems like there is a misunderstanding... I was asking to remove this TODO comment, not the tests.

Reply:
my fault